### PR TITLE
Add iOS11 alert

### PR DESCRIPTION
**Pull request by subsymbolic**
Reviewer: Chris
Asana: https://app.asana.com/0/414235014887631/433059546511658

**Description**:
Add a popup letting our beta testers know that an update for iOS 11 is coming soon

**Steps to test this PR**:
TEST 1
1. Launch the app
2. Note that you see a alert on the main screen

TEST 2
1. Launch the app again
2. Ensure that you do not see the alert a second time

###### Reviewer Checklist:
- [ ] **Ensure the PR solves the problem**
- [ ] **Review every line of code**
- [ ] **Ensure the PR does no harm by testing the changes thoroughly**
- [ ] **Get help if you're uncomfortable with any of the above!**
- [ ] Determine if there are any quick wins that improve the implementation

###### PR DRI Checklist:
- [ ] Get advice or leverage existing code
- [ ] Agree on technical approach with reviewer (if the changes are nuanced)
- [ ] Ensure that there is a testing strategy (and documented non-automated tests)
- [ ] Ensure there is a documented monitoring strategy (if necessary)
- [ ] Consider systems implications (Database connections, Grafana stats, CPU)

### CODE REVIEW

File: DuckDuckGo/MainViewController.swift [hunk 1]
```diff
@@ -64,6 +64,25 @@ class MainViewController: UIViewController {
...
     }
     
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        showiOS11Popup()
+    }
+    
+    private func showiOS11Popup() {
+        var settings = TutorialSettings()
+        if !settings.hasSeenOnboarding, settings.hasSeeniOS11Popup {
```
Comment (brindy) [inline DuckDuckGo/MainViewController.swift]: The popup shows every time the app is restarted.  

I believe this syntax is equivalent to `&&` but you probably want this to be a `||`?

File: DuckDuckGo/MainViewController.swift [hunk 2]
```diff
@@ -64,6 +64,25 @@ class MainViewController: UIViewController {
         }
     }
     
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        showiOS11Popup()
```
Comment (brindy) [inline DuckDuckGo/MainViewController.swift]: No need to show this to iOS 10< users, could wrap it in 

`if #available(iOS 11.0, *) {`

If so, you probably want to update the copy of your alert message as well.

Comment (subsymbolic) [reply DuckDuckGo/MainViewController.swift]: Done! Good idea.

**Comment by subsymbolic** [comment]
Closing as we no longer need this.

**Pull request closed without merging**